### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by dslh**
The builder gem is now required, for some reason. Other than that, there do not seem to be any problems caused by the upgrade. 😄 Please take a look, @gohiring/engineers!

### CODE REVIEW

**Comment by dslh** [comment]
Looks like I spoke too soon. 😆 There are a couple of rubocop things that needed to be addressed, and one development dependency that won't build if rake isn't installed, so I locked it to an earlier version. Ready for review now 👍

File: .rubocop.yml [hunk 1]
```diff
@@ -9,3 +9,6 @@ Style/PredicateName:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+Metrics/BlockLength:
```
Comment (pvcarrera) [inline .rubocop.yml]: What do you think about ignoring the comments and setting a max lenght? I think it's nice to have a warning when methods are too long.
```yml
Metrics/BlockLength:
  CountComments: false
  Max: 25
```

**Comment by dslh** [comment]
Hey @pvcarrera! Please take another look. The problem with the `Metrics/BlockLength` cop is that it doesn't play nicely with rspec. Each `*_spec.rb` file keeps its code in a `describe` block that can run to several hundred lines of code. I though about disabling the `BlockLength` cop for `spec/**/*`, but then I found it has an `ExcludedMethods` setting that seems perfect for our needs. I've disabled the cop for rspec's `context` and `describe` methods, and now everything seems to be fine.

What do you think? Maybe we can apply this same approach to our other projects 😄

**Comment by pvcarrera** [comment]
I really like this idea 👍 . Sure, we should spread this configuration to all our repos, daily hits? 😄

**Pull request merged**